product_docs/docs/pgd/5/sequences.mdx

```diff
@@ -18,6 +18,12 @@ produce values that are unique only on the local node. This is important because
 unique ids generated by such sequences cause conflict and data loss by
 means of discarded `INSERT` actions in multi-master replication.
 
+!!! Note Permissions Required
+PGD global sequences are on by default. This means that for any user to use sequences, 
```

Comment:
Do we have docs that discuss how/when normal sequences are converted to global ones? Perhaps worth linking it here?